internal/pkg/scaffold/ansible/go_mod.go

```diff
@@ -103,7 +59,12 @@ replace (
 	k8s.io/kubernetes => k8s.io/kubernetes v1.13.4
 )
 
-replace github.com/coreos/prometheus-operator => github.com/coreos/prometheus-operator v0.29.0
+replace (
+	github.com/coreos/prometheus-operator => github.com/coreos/prometheus-operator v0.29.0
+	// Pinned to v2.9.2 (kubernetes-1.13.1) so https://proxy.golang.org can
```

Comment:
We're pinned to kubernetes-1.13.4. Does that change the transitive prometheus version?

```suggestion
	// Pinned to v2.9.2 (kubernetes-1.13.4) so https://proxy.golang.org can
```

Reply:
There's no prometheus version that pins dependencies to 1.13.4. The commit for v2.9.2 contains the last pin to a 1.13.x version before their 1.14.x bump (v2.10.0). I believe go mod attempts to resolve to v2.10.0 but technically that is incorrect right?

Reply:
Oh gotcha. So prometheus has a dependency on kubernetes. I was thinking the other way around. Yeah sounds like v2.9.2 is correct.